src/platform_impl/windows/event_loop.rs

```diff
@@ -173,6 +175,7 @@ impl<T: 'static> EventLoop<T> {
                 },
                 _marker: PhantomData,
             },
+            translate_accel_callback: Some(Box::new(|_| false)),
```

Comment:
shouldn't this pass `attributes.translate_accel_callback` here?

Reply:
Right. Silly mistake.